modules/ROOT/pages/security.adoc

```diff
@@ -0,0 +1,252 @@
+|===
+
+|Bootstrap Configuration
+|Command-line flag
+|Default
+|Purpose
+
+|bootstrap.use_tls_server
+|-bootstrap.use_tls_server
+|`true`
+|Default: TLS enabled +
+Set this `false` to allow use of non-secure protocols (`couchbase://` instead of `couchbases://`) -- *development and testing only*
+
+|bootstrap.server_tls_skip_verify
+|-bootstrap.server_tls_skip_verify
+|`false`
+|Default: TLS enabled +
+Set this `true` to skip validation of any certificates received from the server with the CA certificate -- *development and testing only*
+
+|api.https.tls_key_path
+|-api.https.tls_key_path
+|nil
+|Provide the path footnote:[pass:q,a[This can be absolute, or relative to the {sgw} executable's directory]] to the TLS private key file.
+
+Set to `""` or _nil_ to use _plaintext_ -- development or test use *only*
+
+|api.https.tls_cert_path
+|-api.https.tls_cert_path
+|nil
+|Provide the path to the TLS certificate file.
+
+Set to `""` or _nil_ to use _plaintext_
+
+|bootstrap.ca_cert_path
+|-bootstrap.ca_cert_path
+|CA Root Pool
+|Provide the path to the root CA certificate to verify the certificate chain and hostname of the Couchbase Server cluster
+
+Set to `""` or _nil_ to use _plaintext_ -- when `server_tls_skip_verify=true`
+
+|bootstrap.cert_path
+|-bootstrap.cert_path
+|nil
+|Provide the path to the client's certificate to authenticate against Couchbase Server footnote:[5.5 or above]
+
+Set to `""` or _nil_ to use of _plaintext_
+
+|bootstrap:key_path
+|-bootstrap.key_path
+|nil
+|Provide the path to the the client's private key to authenticate against Couchbase Server footnote:[5.5 or above]
+
+Set to `""` or _nil_ to force use of a _plaintext_
+
+|===
+
+
+== Compatibility
+
+Server Compatibility::
++
+--
+By default the server scheme is specified as `couchbases://`.
+
+You can set the {bootstrap-schema--xref--bootstrap-server-tls-skip-verify} flag `true` to connect to 'default CBS'.
+But, this must only be done for testing or development purposes. 
+--
+
+Client Compatibility::
+Couchbase Lite client applications must be updated to use TLS when connecting to {sgw} nodes running in default mode.
+
+
+
+== Behavior
+
+The content in <<tbl-route-behavior>> shows the communications behavior between Sync Gateway and communication endpoints.
+
+
+.Sync Gateway behavior
+[#tbl-route-behavior, cols="4a,6a,4,6a, options="header"]
+|===
+
+.>h|Endpoint
+.>h|Required Config Items
+.>h|TLS Opt-Out Trigger
+.>h|Behavior
+
+|REST API, Couchbase Lite and Inter-sync gateway replication
+|{bootstrap-schema--xref--api-https-tls-cert-path} +
+{bootstrap-schema--xref--api-https-tls-key-path}
+|Provide nil values for the key/certificate paths
+|TLS encryption is enabled by default, unless the opt-out is triggered
+
+|Couchbase Server
+|Use `couchbases://` server scheme +
+Set the certificate or key path:
+
+* {bootstrap-schema--xref--bootstrap-ca-cert-path} +
+* {bootstrap-schema--xref--bootstrap-certpath} +
+* {bootstrap-schema--xref--bootstrap-keypath}
```

Comment:
Also needs removing if not already

Reply:
Done -- https://github.com/couchbase/docs-sync-gateway/pull/541